datamonster_api/lib/datamonster.py

```diff
@@ -433,6 +438,69 @@ def to_json_checked(filters):
                 "Problem with filters when getting dimensions: {}".format(e)
             )
 
+    ##############################################
```

Comment:
worth creating a new class? no strong feelings here

Reply:
I'd say no -- we already have methods for companies and datasources in the same class